lib/src/data/communication/firebase/firebase_authorization.dart

```diff
@@ -0,0 +1,25 @@
+import 'dart:async';
+
+import 'package:allthenews/src/domain/authorization/authorization.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+
+class FirebaseAuthorization implements Authorization {
```

Comment:
W zasadzie to Authentication :) - może FirebaseAuthenticationClient i odpowiednio AuthenticationClient interfejs? Trochę inaczej popakietować można, bo teraz w communication jest pakiet `api` i pakiet `firebase` a w zasadzie oba to api 🤔

Reply:
proponuję to co w api obecnie dać do pakietu `nytimes` i wrzucić paczkę `firebase` do api